spec/views/info/_lead_metrics.html.erb_spec.rb

```diff
@@ -56,7 +56,7 @@
     let(:data) { { unique_pageviews_average: 13245, exits_via_search_average: 12, problem_reports_weekly_average: 33 } }
 
     it "uses multipart metrics" do
-      locals[:@is_multipart] = true
```

Comment:
I am 100% surprised that this ever worked. I would not expect setting a local with an `@` in the name to turn into an instance variable in the view.  These changes are worth it just to get rid of this!